packages/components/src/components/form/Switch/Switch.stories.tsx

```diff
@@ -1,26 +1,13 @@
-import React from 'react';
-import { useArgs } from '@storybook/client-api';
-
+import { StoryObj } from '@storybook/react';
 import { Switch as SwitchComponent } from './Switch';
 
 export default {
     title: 'Form/Switch',
+    component: SwitchComponent,
 };
 
-export const Switch = {
-    render: ({ ...args }) => {
-        // eslint-disable-next-line
-        const [{ isChecked }, updateArgs] = useArgs();
-        const handleIsChecked = () => updateArgs({ isChecked: !isChecked });
-
-        return (
-            <SwitchComponent
-                onChange={handleIsChecked}
-                isChecked={isChecked}
-                isSmall={args.isSmall}
-                isDisabled={args.isDisabled}
-            />
-        );
+export const Switch: StoryObj = {
```

Comment:
Revert the change, please, it's not working on click like this :DD I also fell into that trap when reviewting yesterday.

And for nicer controls you can add:

```
argTypes: {
        labelPosition: {
            options: ['left', 'right'],
            control: {
                type: 'radio',
            },
        },
    },
```

Reply:
9ada0fde38e5e788ad4b7eba036e50f07baef91f